metadata_service/api/swagger_doc/column/badge_put.yml

```diff
@@ -0,0 +1,47 @@
+Add badge to a resource
+---
+tags:
+  - 'table'
+  - 'dashboard'
```

Comment:
i don't know what `tags` mean here, but does this apply?

Reply:
I used the same yml as the table ones, so i just kept them here... not sure what they mean...